airflow/providers/slack/CHANGELOG.rst

```diff
@@ -33,6 +36,27 @@ Breaking changes
 * Drop support deprecated ``webhook_token`` parameter in
   :ref:`Slack Incoming Webhook Connection <howto/connection:slack-incoming-webhook>` Extra.
 
+* ``Refactor 'SlackWebhookOperator': Get rid of mandatory http-provider dependency (#26648)``
+* ``Refactor SlackWebhookHook in order to use 'slack_sdk' instead of HttpHook methods (#26452)``
+* ``Move send_file method into SlackHook (#26118)``
+
+Features
+~~~~~~~~
+
+* ``Refactor Slack API Hook and add Connection (#25852)``
+* ``Add common-sql lower bound for common-sql (#25789)``
+
+Bug Fixes
+~~~~~~~~~
+
+* ``Remove unsafe imports in Slack API Connection (#26459)``
+
```

Comment:
I do know is #26459 should mention in changelog because it fix #25852 ?

Reply:
I put it to "features" together with #25852. 

The nice thing of having those "code impacting" changes is that you (and users) can find them in the changelog and we can always look up the code change.